packages/commonwealth/client/scripts/views/modals/edit_topic_thresholds_modal.tsx

```diff
@@ -11,16 +11,23 @@ import { TokenDecimalInput } from 'views/components/token_decimal_input';
 import { CWButton } from '../components/component_kit/cw_button';
 import { CWText } from '../components/component_kit/cw_text';
 import { CWIconButton } from '../components/component_kit/cw_icon_button';
+import {
+  useFetchTopicsQuery,
+  useSetTopicThresholdMutation,
+} from 'state/api/topics';
 
 type EditTopicThresholdsRowProps = {
   topic: Topic;
+  onClose: () => void;
 };
 
-const EditTopicThresholdsRow = (props: EditTopicThresholdsRowProps) => {
+const EditTopicThresholdsRow = ({
+  topic,
+  onClose,
+}: EditTopicThresholdsRowProps) => {
   const [newTokenThresholdInWei, setNewTokenThresholdInWei] =
     React.useState<string>();
-
-  const { topic } = props;
+  const { mutateAsync: setTopicThreshold } = useSetTopicThresholdMutation();
```

Comment:
i think we should destructure the `error`, `data` values to show error/success status